### PR TITLE
Category wikis

### PR DESCRIPTION
**Pull request by Emmanueldmlr**
fixes https://github.com/EveripediaNetwork/issues/issues/147

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,6 +47,7 @@
     "ipfs-http-client": "55.0.0",
     "ismobilejs": "^1.1.1",
     "match-sorter": "^6.3.1",
+    "moment": "^2.29.1",
```
Comment (kesar) [inline package.json:50]: https://blog.bitsrc.io/date-fns-vs-momentjs-9833f7463751

pls use date fns if possible

Comment (Emmanueldmlr) [reply package.json:50]: Okay then, I will update the codebase and the package.json file. Thanks for the resource, what an eye-opener.

File: src/components/Categories/SubCategoryCard.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      <Box
+        w={390}
+        minH={390}
+        boxShadow="2xl"
+        rounded="md"
+        p={6}
+        overflow="hidden"
+      >
+        <Box h={200} mb={3} pos="relative">
+          <Image src="/images/sub-category-image.png" layout="fill" />
```
Comment (kesar) [inline src/components/Categories/SubCategoryCard.tsx:22]: image must come from wiki and default to a default one if image returns 404

Comment (Emmanueldmlr) [reply src/components/Categories/SubCategoryCard.tsx:22]: image
Images array doesn't have an Image property on it

Comment (kesar) [reply src/components/Categories/SubCategoryCard.tsx:22]: https://gateway.pinata.cloud/ipfs/ + id

first part of the url i think its an env / config var

Comment (Emmanueldmlr) [reply src/components/Categories/SubCategoryCard.tsx:22]: Wao...I didn't know about this. Thanks. I will implement it now.

**Review by kesar — APPROVED** [review]

**Pull request merged**